boat_simulator/nodes/physics_engine/fluid_forces.py

```diff
@@ -50,11 +82,73 @@ def compute(self, apparent_velocity: NDArray, orientation: Scalar) -> Tuple[NDAr
         Returns:
             Tuple[NDArray, NDArray]: A tuple containing the lift force and drag force experienced
                 by the medium, both expressed in newtons (N).
+                Bound to 360 degrees before using.
         """
 
-        # TODO: Implement this method.
+        attack_angle = self.calculate_attack_angle(apparent_velocity, orientation)
+        lift_coefficient, drag_coefficient, area = self.interpolate(attack_angle)
```

Comment:
When switching to a symmetric range, remember to pass in the absolute angle of attack since this function seems to want that.